src/three-render-objects.js

```diff
@@ -43,11 +43,9 @@ const three = window.THREE
   Clock
 };
 
-import ThreeTrackballControls from 'three-trackballcontrols';
-import OrbitControlsWrapper from 'three-orbit-controls';
-const ThreeOrbitControls = OrbitControlsWrapper(three);
-import FlyControlsWrapper from 'three-fly-controls';
-const ThreeFlyControls = (FlyControlsWrapper(three), three.FlyControls);
+import { TrackballControls as ThreeTrackballControls } from 'three/examples/jsm/controls/TrackballControls.js';
+import { OrbitControls as ThreeOrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
```

Comment:
can't you just import from `three`?  not the example code?

Reply:
I tried to, but node couldn't find the object. The only part of the code where the controls are exported are on the example folder.

Also I did a fast research and found that [on the threejs examples](https://github.com/mrdoob/three.js/blob/master/examples/misc_controls_fly.html#L36) they export them from that path.

Reply:
wow looks like it, opened an issue here: https://github.com/mrdoob/three.js/issues/19250